### PR TITLE
zh: add istio ca translation and license

### PR DESCRIPTION
**Pull request by rootsongjc**
- Add Istio CA documentation translation
- Add Apache 2 license

### CODE REVIEW

**Comment by rootsongjc** [comment]
## Links error

- https://deploy-preview-70--getistio-demo.netlify.app/zh/getistio-cli/reference/getistio 404
- https://deploy-preview-70--getistio-demo.netlify.app/getistio-cli/reference/getistio_istioctl/ No corresponding Chinese URL is generated.

**Comment by lunchboxav** [comment]
@rootsongjc do you know how the chinese translation is generated? I may need to learn about this before solving the missing links

**Comment by rootsongjc** [comment]
> @rootsongjc do you know how the Chinese translation is generated? I may need to learn about this before solving the missing links

The English documentation is located in the `content/en` directory. The Chinese documentation is located in the `content/zh` directory. Just copy from the English folder.

File: layouts/partials/default.html [hunk 1]
```diff
@@ -79,7 +79,7 @@ <h2 class="mb-4 font-weight-medium">{{ .Title }}</h2>
 {{ with .File }}{{ $fileUniqueID = .UniqueID }}{{ end }}
 {{ $currentNodeFileUniqueID := "" }}
 {{ with $currentNode.File }}{{ $currentNodeFileUniqueID = .UniqueID }}{{ end }}
-<li data-nav-id="{{.Permalink}}" title="{{.Title}}"
+<li data-nav-id="{{.Permalink | relURL }}" title="{{.Title}}"
```
Comment (rootsongjc) [inline layouts/partials/default.html:82]: I think the problem is in the template, can you look at it? @lunchboxav

Comment (lunchboxav) [reply layouts/partials/default.html:82]: ow I see.. ok I'll take a look. thanks @rootsongjc

**Comment by lunchboxav** [comment]
I think I know the culprit, the getIstio CLI reference use a refresh, but it doesn't work as expected. I'll look into this.

Also, @rootsongjc is this PR dependant on those 2 broken links that you mention? If not, you can merge this PR and I'll fix those links in different PR.

**Comment by rootsongjc** [comment]
> I think I know the culprit, the getIstio CLI reference use a refresh, but it doesn't work as expected. I'll look into this.
> 
> Also, @rootsongjc is this PR dependant on those 2 broken links that you mention? If not, you can merge this PR and I'll fix those links in different PR.

@lunchboxav I think we can merge this PR first, the links that I boxed in the picture are problematic and you can fix them in the next PR.

**Comment by lunchboxav** [comment]
yes, pls do @rootsongjc, don't forget to resolve the conflicts, I'll file the fix in different PR

**Pull request merged**